core/arch/arm/tee/entry_std.c

```diff
@@ -113,7 +113,7 @@ static TEE_Result copy_in_params(const struct optee_msg_param *params,
 {
 	TEE_Result res;
 	size_t n;
-	uint8_t pt[TEE_NUM_PARAMS];
+	uint8_t pt[TEE_NUM_PARAMS] = {0};
```

Comment:
Nit: Please add space before and after the `0`.

Reply:
👍 sorry, should've run `checkpatch`, it's been a rough morning